vim.js

```diff
@@ -16,6 +17,12 @@ document.addEventListener('keypress', function(evt){
     if (evt.key == 'k') {
       window.scrollByLines(-1);
     }
+    if (evt.key == 'g') {
+      gState = "GOTO";
```

Comment:
We might need to go back to NORMAL mode when we click ESC. Otherwise the user may get stucked in the GOTO mode